src/index.html

```diff
@@ -11,7 +11,212 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+      class="form"
+      method="post"
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+        <div class="form-field">
+
```

Comment:
Like Radek mentioned before there should not be an empty line.

"You should have empty lines only between siblings on the same level, not between parent and child, here's an example"

```
<section>
  <div>
    <span>First Child of Div</span>

    <span>Second Child of Div</span>
  <div>
</section>
```